### PR TITLE
Allow breaking out strings into two translations

### PR DESCRIPTION
**Pull request by yiqunw700**
Add an icon button in the UI for this function. (Couldn't make the icon-button in the same line with the checkbox and the node id though...).

Action is not saved... Will look into it in a following pr.
Towards #1469

### CODE REVIEW

File: support-figma/extended-layout-plugin/src/localization-module.ts [hunk 1]
```diff
@@ -134,6 +134,77 @@ export async function updateStringRes(strRes: StringResource) {
...
+        translatable: true,
+        text: strRes.text,
+        textNodes: [{ nodeId: node.id, isExcluded: isNodeExcluded }],
+        extras: strRes.extras,
+        textLength: strRes.textLength,
+      };
+      stringResourceMap.set(stringResName, newStrRes);
+
+      // Convert to an array of key-value pairs
+      const stringReourceArray = Array.from(stringResourceMap);
```
Comment (rylin8) [inline support-figma/extended-layout-plugin/src/localization-module.ts:189]: speling: 'stringReourceArray'

File: support-figma/extended-layout-plugin/src/ui.html [hunk 2]
```diff
@@ -937,8 +938,32 @@
...
+      editBtn.onclick = () => {
+        parent.postMessage({
+          pluginMessage: {
+            msg: 'localization-ungroup-node',
+            node: node["nodeId"],
+            stringResMap: Array.from(outputStringData)
+          }
+        }, '*');
+      };
+      block.append(editBtn);
```
Comment (rylin8) [inline support-figma/extended-layout-plugin/src/ui.html:964]: When I mouse over this button, it changes to a hand icon for 1-2 pixels on the perimeter of the button, then back to an arrow in the center. I think it's changing to the hand when it's over the 'icon-button' but not over the 'icon--break' icon. Can you fix this so it's either the arrow or the hand icon the whole time? I also think a short tooltip would help, even though there are instructions at the top.

**Review by rylin8 — COMMENTED** [review]
What's the best way to "Undo" this action, or to combine similar text nodes to the same string resource label?

File: support-figma/extended-layout-plugin/src/localization-module.ts (hunk 1 above)
Comment (yiqunw700) [reply support-figma/extended-layout-plugin/src/localization-module.ts]: Done.

File: support-figma/extended-layout-plugin/src/ui.html (hunk 2 above)
Comment (yiqunw700) [reply support-figma/extended-layout-plugin/src/ui.html:979]: Done.

**Comment by yiqunw700** [comment]
> What's the best way to "Undo" this action, or to combine similar text nodes to the same string resource label?

Yes. To undo it, rename the resource label to the one it would like to merge with.

**Review by rylin8 — APPROVED** [review]

**Pull request merged**